src/python/WMCore/MicroService/MSUnmerged/MSUnmerged.py

```diff
@@ -185,12 +187,24 @@ def execute(self):
         try:
             self.protectedLFNs = set(self.wmstatsSvc.getProtectedLFNs())
             # self.logger.debug("protectedLFNs: %s", pformat(self.protectedLFNs))
-
             if not self.protectedLFNs:
-                msg = "Could not fetch the protectedLFNs list from WMStatServer. "
+                msg = "Could not fetch the protectedLFNs list from Production WMStatServer. "
                 msg += "Skipping the current run."
                 self.logger.error(msg)
                 return summary
+
+            self.protectedLFNsT0 = set()
+            if self.msConfig['enableT0WMStats']:
+                self.protectedLFNsT0 |= set(self.wmstatsSvcT0.getProtectedLFNsT0())
+                # self.logger.debug("protectedLFNsT0: %s", pformat(self.protectedLFNsT0))
+                if not self.protectedLFNsT0:
+                    msg = "Could not fetch the protectedLFNs list from T0 WMStatServer. "
+                    msg += "Skipping the current run."
+                    self.logger.error(msg)
+                    return summary
+
+            self.protectedLFNs |= self.protectedLFNsT0
```

Comment:
IMO, this `|=` operator is easy to miss when reading this code. So I would suggest to expand it to:
```
            self.protectedLFNs = self.protectedLFNs | self.protectedLFNsT0
```

It's a matter of taste though, if you prefer to keep the current code, that's fine as well.

Reply:
yes, I agree it is a little bit more readable. I'll change that one too.